src/GCHP_GridComp/GCHPctmEnv_GridComp/GCHPctmEnv_GridCompMod.F90

```diff
@@ -579,23 +548,20 @@ subroutine Run ( GC, IMPORT, EXPORT, CLOCK, RC )
       ENDIF
 #endif
 
-      ! Get to the imports...
-      ! ---------------------
-      call MAPL_GetPointer ( IMPORT,     PS0,    'PS1', RC=STATUS )
-      _VERIFY(STATUS)
-      call MAPL_GetPointer ( IMPORT,     PS1,    'PS2', RC=STATUS )
-      _VERIFY(STATUS)
-      call MAPL_GetPointer ( IMPORT,      UA,    'UA',  RC=STATUS )
+      !! Calculate PLE0r8, PLE1r8, DryPLE0r8, and DryPLE1r8 exports
+      call lgr%debug('Calculating PLE[01]r8 and DryPLE[01]r8 exports.')
+
+      ! Imports 
+      call MAPL_GetPointer( IMPORT, PS0, 'PS1', RC=STATUS )
       _VERIFY(STATUS)
-      call MAPL_GetPointer ( IMPORT,      VA,    'VA',  RC=STATUS )
+      call MAPL_GetPointer( IMPORT, PS1, 'PS2', RC=STATUS )
```

Comment:
Why is there a mismatch in naming between the import name and the local pointer? SPHU has suffix _IMPORT. How about using that for all the pointers to imports and keeping the prefix the same as the import name?

Reply:
Just legacy code. I'm in favor of the `_IMPORT` suffix. I'll add that

Reply:
Added in #181 (merged into this PR)